connectors/src/connectors/confluence/index.ts

```diff
@@ -44,7 +44,12 @@ import {
   BaseConnectorManager,
   ConnectorManagerError,
 } from "@connectors/connectors/interface";
+import { dataSourceConfigFromConnector } from "@connectors/lib/api/data_source_config";
 import { concurrentExecutor } from "@connectors/lib/async_utils";
+import {
```

Comment:
Just checking: it seems there are multiple levels in confluence hierarchy
cf picture: test > test 123 > some files
Are those subspaces ? are those properly taken into account?
![image](https://github.com/user-attachments/assets/3b541afe-fea9-4115-8735-c7b68dd4fe9e)

Reply:
these are pages, which are not folders but can be parents to other pages

Reply:
as per IRL: we're good 👍